Makefile

```diff
@@ -63,19 +71,24 @@ ifeq (,$(wildcard $(GOPATH)/src/$(VDDK_LIBS)))
 	$(error "$(GOPATH)/src/$(VDDK_LIBS) cannot find vddk libs in path, please reference to: https://github.com/vmware-tanzu/astrolabe/tree/master/vendor/github.com/vmware/gvddk#dependency")
 endif
 
-plugin: datamgr
+plugin: datamgr backupdriver
 	@echo "making: $@"
-	$(MAKE) build BIN=$(PLUGIN_BIN) VERSION=$(VERSION)
+	$(MAKE) build BIN=$(PLUGIN_BIN) INP=$(PLUGIN_INP) VERSION=$(VERSION)
 
 datamgr: astrolabe
 	@echo "making: $@"
-	$(MAKE) build BIN=$(DATAMGR_BIN) VERSION=$(VERSION)
+	$(MAKE) build BIN=$(DATAMGR_BIN) INP=$(DATAMGR_INP) VERSION=$(VERSION)
+
+backupdriver:
+	@echo "making: $@"
+	$(MAKE) build BIN=$(BACKUPDRIVER_BIN) INP=$(BACKUPDRIVER_INP) VERSION=$(VERSION)
```

Comment:
No need to add `INP` in all of the three places if you follow the suggestion above.